.kitchen.yml

```diff
@@ -206,6 +206,31 @@ suites:
       - recipe[hadoop::zookeeper]
       - recipe[hadoop::zookeeper_server]
     attributes: { hadoop: { distribution: 'bigtop', distribution_version: '0.8.0' } }
+  - name: bigtop100
+    excludes: debian-6.0.10, ubuntu-14.04
```

Comment:
Looks like Ubuntu 14 is supported, but not Ubuntu 12... Have you tested this on Ubuntu 12?